clang/tools/scan-view/bin/scan-view

```diff
@@ -39,7 +36,6 @@ def url_is_up(url):
 
 
 def start_browser(port, options):
-    import webbrowser
```

Comment:
Due to this import, the import on L17 is unused.

let me know if keeping this one and removing the import from L17 is more preferable.